roles/forkliftcontroller/defaults/main.yml

```diff
@@ -30,10 +30,10 @@ inventory_volume_path: "/var/cache/inventory"
 inventory_container_name: "{{ app_name }}-inventory"
 inventory_service_name: "{{ app_name }}-inventory"
 inventory_route_name: "{{ inventory_service_name }}"
-inventory_container_limits_cpu: "10000m"
-inventory_container_limits_memory: "8Gi"
-inventory_container_requests_cpu: "1000m"
-inventory_container_requests_memory: "2Gi"
+inventory_container_limits_cpu: "1000m"
```

Comment:
This will be applied as `"1"` instead of `"1000m"` so could have specified it here.